Cookbook/Technical-Documents/From_0%_to_100%_with_Bento.md

```diff
@@ -0,0 +1,38 @@
+ Outline of "From 0% to 100% screen with Bento"
+
+## Why & when to use Bento?
+I'd like to inform the reader as to why developing a screen with Bento will take less time than writing it from scratch.
+
+### Examples from our code
+Provide examples of our app & code what screens are written in Bento and link them here as a reference point 
+
+### Bento & SwiftUI
+Small explanation on what our team's current approach to adopting SwiftUI looks like. This will be a be a short section saying that Bento will eventually replaced by SwiftUI (in few years perspective).
+
+## How to use BabylonBoxViewController, ViewModel & Renderer
+Section when I'd like to describe how to setup a typical RAF module with strong focus on `Renderer` part.
+I want this section to be a guide with code samples, line by line, how to write a renderer.
+
+### Keywords in Bento
+- What are `Screen`, `Box`, `Section`, `Node`.
+- What is a `BabylonBoxViewController`?
+- What is `AnyRenderable`
+- What Operators can I use?
+
+## Writing a custom component (optional)
```

Comment:
This is an important section, it shouldn't be optional. It should also serve as a mental model for any person writting UI code:

1. Check if the component exists
  1. Yes: Ok, use it via Design Library. How should I check if the component is there or not? 
  2. No: Ok, create one and add it to the Design Library. If it's not there, why is that the case? Is this something completely new? Who should I inform when this is done?

Reply:
Ok. At first I was thinking about something else but this is a problem I can write about.
I'll change the section title to:

## Writing a component which hasn't been implemented yet 

and it won't be optional

Reply:
How about

## Creating a new component ## ?